tests/test_config.py

```diff
@@ -0,0 +1,186 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Oxford Quantum Circuits Ltd
+from os.path import abspath, dirname, join
+from sys import __loader__
+
+import pytest
+
+from compiler_config.config import (
+    CompilerConfig,
+    InlineResultsProcessing,
+    MetricsType,
+    OptimizationConfig,
+    Qasm2Optimizations,
+    QiskitOptimizations,
+    QuantumResultsFormat,
+    ResultsFormatting,
+    TketOptimizations,
+)
+
+SUPPORTED_CONFIG_VERSIONS = ["v02", "v01", "v1"]
+
+
+class TestType:
+    pass
+
+
+def _get_json_path(file_name):
+    return join(
+        abspath(join(dirname(__file__), "serialised_compiler_config_templates", file_name))
+    )
+
+
+def _get_contents(file_path):
+    """Get Json from a file."""
+    with open(_get_json_path(file_path)) as ifile:
+        return ifile.read()
+
+
+class TestConfigGeneral:
```

Comment:
What is the class adding here (other than 4 spaces at the start of every following line and "self" in every method)? I would remove it

Reply:
yeah fair, removed